beater/server.go

```diff
@@ -70,25 +79,22 @@ func run(server *http.Server, lis net.Listener, config *Config) error {
 	case true:
 		logger.Info("RUM endpoints enabled!")
 	case false:
-		logger.Info("RUM endpoints disabled")
+		logger.Info("RUM endpoints disabled.")
 	}
 
 	if config.MaxConnections > 0 {
 		lis = netutil.LimitListener(lis, config.MaxConnections)
-		logger.Infof("connections limit set to: %d", config.MaxConnections)
+		logger.Infof("Connections limit set to: %d", config.MaxConnections)
 	}
 
-	ssl := config.SSL
-	if ssl.isEnabled() {
-		cert, err := outputs.LoadCertificate(&config.SSL.Certificate)
-		if err != nil {
-			return err
-		}
-		server.TLSConfig = &tls.Config{Certificates: []tls.Certificate{*cert}}
+	if server.TLSConfig != nil {
+		logger.Info("SSL enabled.")
 		return server.ServeTLS(lis, "", "")
 	}
 	if config.SecretToken != "" {
 		logger.Warn("Secret token is set, but SSL is not enabled.")
+	} else {
+		logger.Info("SSL disabled.")
```

Comment:
won't this log the same as line 91?  I think it can be removed

Reply:
91 is logging `SSL enabled`, here I log `SSL disabled` to ensure both cases are covered in the logs.

Reply:
ah 🤦‍♂